RestaurantReservationSystem/EntityServices/CustomerService.cs

```diff
@@ -0,0 +1,96 @@
+﻿using RestaurantReservation.Db.Models;
+using Microsoft.EntityFrameworkCore;
+
+public class CustomerService
+{
+    private readonly CustomerOperations _customerOperations;
+
+    public CustomerService(CustomerOperations customerOperations)
+    {
+        _customerOperations = customerOperations;
+    }
+
+    public async Task AddCustomerAsync()
+    {
+        var customer = new Customer
```

Comment:
instead of hardcoding all of these values, lets pass them to the methods and return them instead of printing them to the screen. That's how you will be using the services later, you have a service method either accepts input, returns values or do some operation and returns nothing.

Reply:
lets apply this comment to all services.